addons/payment_buckaroo/models/payment_provider.py

```diff
@@ -6,7 +6,7 @@
 
 from odoo import fields, models
 
-from odoo.addons.payment_buckaroo.const import SUPPORTED_CURRENCIES
+from odoo.addons.payment_buckaroo.const import SUPPORTED_CURRENCIES, DEFAULT_PAYMENT_METHODS_CODES
```

Comment:
Import only `const` and replace usages of `SUPPORTED_CURRENCIES` by `const.SUPPORTED_CURRENCIES`.